### ISSUE TITLE
[FEATURE] Add cross-faction players to group via nameplate menu options

### ISSUE THREAD

**Issue by gitdalisar** [gitdalisar/mod-Faction-Free#10]
**Is your feature request related to a problem? Please describe.**
Not related to a problem or bug, this is something again tied to the cross-faction settings in the worldserver.conf file. Will need to see if there is something either within .cpp files or maybe this is a UI .lua file change that needs to occur.

**Describe the solution you'd like**
Same functionality as if same faction, right-click a person's profile and have the option for "invite to group"

**Describe alternatives you've considered**
Nothing at this point. The only thing you can do is /invite command to get a person in or invite if in guild or friends list.

**Additional context**
Initially reported by Ghost vi DM on Discord.

**Comment by gitdalisar** [comment]
I found a way to do this as well as including whisper. However, I did so with an addon approach and that breaks the ability to set and clear focus from right click menu. This is because Blizzard explicitly prohibits addons from doing any targeting, including focus targeting. So I will need to either find a way to modify (instead of replace) the UnitPopUp menu or go at this in another direction.

**Comment by gitdalisar** [comment]
I was able to modify the UnitPopup Menu to include both whisper and invite. I did so using an addon, which means that this feature is not dependent on this module. That said, I created it as its own repo so that anyone can grab just that alone. Information on what that addon does and the source can be found at: [https://github.com/gitdalisar/addon-Faction-Free-Unit-Popup](https://github.com/gitdalisar/addon-Faction-Free-Unit-Popup)
